<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZEDObjectBrowser</class>
 <widget class="QWidget" name="ZEDObjectBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="txtSearch">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Search Objects</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnMode">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ZEDCommon.qrc">
         <normaloff>:/Icons/List.png</normaloff>
         <normalon>:/Icons/Tree.png</normalon>:/Icons/List.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="trwObjects"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
       <property name="icon">
        <iconset resource="ZEDCommon.qrc">
         <normaloff>:/Icons/Delete.png</normaloff>:/Icons/Delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="icon">
        <iconset resource="ZEDCommon.qrc">
         <normaloff>:/Icons/Add.png</normaloff>:/Icons/Add.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ZEDCommon.qrc"/>
 </resources>
 <connections/>
</ui>
